### DISCUSSION TITLE
is it possible hide client 127.0.0.1

### DISCUSSION THREAD

**Discussion by Yumega** [AdguardTeam/AdGuardHome#7198] [General]
is it possible to hide logs from localhost (127.0.0.1)?

**Comment by he-pennypacker** [comment]
I think you can click through Settings > General settings. Under the Logs and Statistics sections, try adding 127.0.0.1 to the ignored domains. I don't have 127.0.0.1 in my logs as I am using the docker host network driver (I get 192.168.1.1 in my logs instead).
